picard/util/cdrom.py

```diff
@@ -63,67 +55,76 @@
 
 LINUX_CDROM_INFO = '/proc/sys/dev/cdrom/info'
 
-# if get_cdrom_drives() lists ALL drives available on the machine
+
+def _generic_iter_drives():
+    config = get_config()
+    yield from (
+        device.strip() for device
+        in config.setting["cd_lookup_device"].split(",")
+        if device and not device.isspace()
+    )
+
+
+def _parse_linux_cdrom_info(f):
+    drive_names = []
+    drive_audio_caps = []
+    while True:
+        line = f.readline()
+        if not line:
+            break
+        if ":" in line:
+            key, values = line.split(':')
```

Comment:
Shouldn't the split here limited?

```python
>>> "a:b:c".split(':')
['a', 'b', 'c']
>>> "a:b:c".split(':', 1)
['a', 'b:c']
```

I know it shouldn't happen in the input, but it's more correct (we expect to split the line in two parts on first `:`).